plugins/in_node_exporter_metrics/ne_cpu_linux.c

```diff
@@ -223,9 +241,15 @@ static int stat_line(char *line, struct cpu_stat_info *st)
                  &st->steal,
                  &st->guest,
                  &st->guest_nice);
-    if (ret != 10) {
+
+    /* On some older kernels the 'guest_nice' value may be missing */
+    if (ret < 9) {
```

Comment:
Not sure if we want to handle future updates where `ret >` 10 by just ignoring that? A future kernel update may add more fields and this would allow us to read the ones we expect plus ignore the rest. However we may want:
```suggestion
    if (ret < 9 || ret > 10) {
```